components/references.bib

```diff
@@ -180,6 +180,13 @@ @manual{gene-id-annotation-rna-seq
   url    = {https://github.com/AlexsLemonade/refinebio-examples/blob/master/03-rnaseq/gene-id-convert_rnaseq_01_ensembl.html}
 }
 
+@website{gene-set-testing-rnaseq,
+    author = {Stephane Ballereau and Mark Dunning and Abbi Edwards and Oscar Rueda and Ashley Sawle},
+    title = {{RNA-seq analysis in R: Gene Set Testing for RNA-seq}},
```

Comment:
Let the algorithm handle capitalization for most things, (keep override for RNA-seq and R)
```suggestion
    title = {{RNA-seq} analysis in {R}: Gene Set Testing for {RNA-seq}},
```

Reply:
Might be a good item for CONTRIBUTING.md (because I looked at it for this info)

Reply:
It would be. Bibtex is strange though... it may be hard to summarize the rules.

Reply:
I don't think we're going to successfully summarize all the rules, just the ones that come up in review.